### ISSUE TITLE
Python version

### ISSUE THREAD

**Issue by zxdawn** [CohenBerkeleyLab/BEHRDownloader#1]
It's better to specify that this only support Python2 now.
Here's the error using Python3:
```
File "BEHRDownloader/behrdownloader/dash_interface.py", line 130, in driver
    if verbose > 0:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

**Comment by joshua-laughner** [comment]
Please provide steps to reproduce

**Comment by zxdawn** [comment]
**Code**
```
./getbehr.sh dash daily-gridded 2014-05 2014-05
```

**Error**
```
Traceback (most recent call last):
  File "/home/xin/Software/anaconda3/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/xin/Software/anaconda3/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/xin/Documents/Github/BEHRDownloader/behrdownloader/getbehr.py", line 24, in <module>
    parse_args()
  File "/home/xin/Documents/Github/BEHRDownloader/behrdownloader/getbehr.py", line 20, in parse_args
    args.driver_fxn(**vars(args))
  File "/home/xin/Documents/Github/BEHRDownloader/behrdownloader/dash_interface.py", line 130, in driver
    if verbose > 0:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

Python version: 3.6.6

**Comment by joshua-laughner** [comment]
Corrected in 7f4f891. `verbose` now correctly defaults to `0` instead of `None` if no "-v" or "--verbose" flags specified. Tested with Python versions 2.7.12 and 3.7.0. Please confirm it works for you now.

**Comment by zxdawn** [comment]
Thanks, it works now.
